tests/setup.js

```diff
@@ -8,7 +8,7 @@ import '@testing-library/jest-dom';
 
 import './redux-persist_mock';
 import './react-intl_mock';
-import './react-router-dom_mock';
+import './react-router-dom_mock.ts';
```

Comment:
I don’t think this is required If I am not wrong we by default search for ts file.

Reply:
it is needed, since importing `.ts` in `.js` files and vice versa need the extension